### DISCUSSION TITLE
PRISMA alternatives? DrizzleORM?

### DISCUSSION THREAD

**Discussion by mehdi-cit** [wasp-lang/wasp#1705] [Archived]
Hello,
Should one expect alternatives to PRISMA when it comes to data access? Personally, what holding me off of WASP-LANG is the lack of support for DRIZZLE ORM.
Thank you for this promising piece of work!

**Comment by Martinsos** [comment]
Thanks @mehdi-cit ! That is a good question -> it is certainly doable in the future, and we are considering it, but it not yet on the roadmap as we have other more urgent priorities.
Meaning I can't give any estimate on it or promise it will happen, but it might. Almost certainly not this year though.
